runtime/pom.xml

```diff
@@ -184,6 +184,12 @@
             <artifactId>commons-collections</artifactId>
             <version>3.2.2</version>
         </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-core</artifactId>
+            <version>1.5.5</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Is this required?

Reply:
Reverted since it was already there in commons package, Thanks.